Makefile.am

```diff
@@ -753,7 +753,7 @@ utils/mkfs/m0mkfs.o: xcode/protocol_checksum.h
 sbin_PROGRAMS                  += utils/layout/m0layout
 utils_layout_m0layout_CPPFLAGS  = -DM0_TARGET='m0layout' $(AM_CPPFLAGS)
 utils_layout_m0layout_LDADD     = $(top_builddir)/motr/libmotr.la \
-                                  @MATH_LIBS@
+                                  @MATH_LIBS@ @ISAL_LIBS@
```

Comment:
Why is the `@ISAL_LIBS@` needed for this utility?
Why is it not needed for others?

I restored it and it builds successfully.

Reply:
The same for desim_ut_chs_test_LDADD

Reply:
I have verified that without the "@ISAL_LIBS@", these two targets get built successfully.

Reply:
Removed.